common/controller_proxy.cpp

```diff
@@ -0,0 +1,102 @@
+/// Copyright 2016 Pinterest Inc.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+/// http://www.apache.org/licenses/LICENSE-2.0
+
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+//
+// @author shu (shu@pinterest.com)
+//
+
+#include "common/availability_zone.h"
+#include "common/network_util.h"
+
+#include <folly/ScopeGuard.h>
+#include <folly/String.h>
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include <cstdio>
+#include <algorithm>
```

Comment:
swap the order of <cstdio> and  <algorithm>?

Reply:
sure